src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.ts

```diff
@@ -102,6 +107,22 @@ export class TextblockFeedbackEditorComponent implements AfterViewInit {
         setTimeout(() => this.textareaAutogrow());
     }
 
+    /**
+     * Set the color of the draggable grading instruction based on the credits of the instruction
+     *  @param {GradingInstruction} instr - the instruction object we set its color based on its credits
+     */
+    setInstrColour(instr: GradingInstruction) {
+        let colour;
+        if (instr.credits === 0) {
+            colour = '#fff2cc';
+        } else if (instr.credits < 0) {
+            colour = '#fbe5d6';
+        } else {
+            colour = '#e3f0da';
+        }
+        return colour;
+    }
```

Comment:
I think get is the better prefix here since no attributes are changed when calling this method. I also think that these colors should be some constants (maybe there are even constants for this?).
```suggestion
    getInstrColour(instr: GradingInstruction): string | undefined {
        if (instr.credits === 0) {
            return '#fff2cc';
        } else if (instr.credits < 0) {
            return '#fbe5d6';
        } else {
            return '#e3f0da';
        }
        return undefined;
    }
```

Reply:
Yes your suggestion with 'get' makes sense, I will adapt it. I think it's okay to leave the colors like this, there are no specific constants.